wondrous-app/components/Table/MembershipRequests.tsx

```diff
@@ -163,6 +107,7 @@ export function MembershipRequestTable(props) {
                 {request?.podName ? (
                   // eslint-disable-next-line @next/next/no-img-element
                   <Link passHref href={`/pod/${request?.podId}/boards`}>
+                    {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
If there is no clear solution yet, we may leave the error instead of disabling it completely. We can return to this if we find a better solution.